tests/acceptance/guid-node/forks-test.ts

```diff
@@ -0,0 +1,146 @@
+        });
+        server.create('contributor', { node, users: contributorUser });
+        const fork = forkNode(
+            server,
+            node as ModelInstance<Node>,
+            {
+                currentUserPermissions: Object.values(Permission),
+            },
+        );
+        const url = `/${node.id}/forks`;
+
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+        assert.dom('[data-test-new-fork-button]').exists({ count: 1 });
+        assert.dom('[data-test-node-card]').exists({ count: 1 });
+        assert.dom('[data-test-node-card-heading]').includesText(fork.title);
+        assert.dom('[data-test-node-menu]').exists({ count: 1 });
+    });
+
+    test('logged in admin, 12 forks', async assert => {
+        const contributorUser = server.create('user', 'loggedIn');
+        const node = server.create<Node>('node', {
+            id: 'f0rk5',
+            title: 'Test Title',
+            currentUserPermissions: [Permission.Admin],
+        });
+        server.create('contributor', { node, users: contributorUser });
+        for (let i = 0; i < 12; i++) {
+            forkNode(
+                server,
+                node as ModelInstance<Node>,
+                {
+                    currentUserPermissions: Object.values(Permission),
+                },
+            );
+        }
+        const url = `/${node.id}/forks`;
+
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+        assert.dom('[data-test-new-fork-button]').exists({ count: 1 });
+        assert.dom('[data-test-node-card]').exists({ count: 10 });
+        assert.dom('[data-test-node-card]').includesText(node.title);
+        await percySnapshot(assert);
+
+        await click('[data-analytics-name="Pagination next"]');
+        assert.dom('[data-test-node-card]').exists({ count: 2 });
+        assert.dom('[data-test-node-card]').includesText(node.title);
+    });
+
+    test('logged in admin, new fork', async assert => {
+        server.create('user', 'loggedIn');
+        const node = server.create<Node>(
+            'node',
+            {
+                id: 'f0rk5',
+                currentUserPermissions: [Permission.Admin],
+            },
+        );
+        const url = `/${node.id}/forks`;
+
+        await visit(url);
+        assert.equal(currentURL(), url, `We are on ${url}`);
+
+        await click('[data-test-new-fork-button]');
+        await percySnapshot(assert);
+        assert.dom('[data-test-close-create-forks-modal]').exists();
+        assert.dom('[data-test-new-fork-modal-body] h3').includesText('Are you sure');
+
+        await click('[data-test-cancel-create-fork]');
+        assert.dom('[data-test-new-registration-modal-body]').doesNotExist();
+
+        await click('[data-test-new-fork-button]');
+        assert.dom('[data-test-confirm-create-fork]').exists();
+        await click('[data-test-confirm-create-fork]');
```

Comment:
Would be good to mock the mirage create fork call and assert that it was called with the correct parameters.